pkgs/tools/audio/volctl/default.nix

```
@@ -2,13 +2,13 @@
 
 pythonPackages.buildPythonApplication rec {
   pname = "volctl";
-  version = "0.6.3";
+  version = "0.8.0";
 
   src = fetchFromGitHub {
     owner = "buzz";
     repo = pname;
     rev = version;
```

The tag names in the upstream repo starts with a `v` since version 0.7.0, so this download doesn't work. See the list of tags: https://github.com/buzz/volctl/releases

```suggestion
    rev = "v${version}";
```